templates/tfengine/recipes/org/foundation.yaml

```diff
@@ -28,8 +28,13 @@ templates:
 {{end}}
 {{if has . "CICD"}}
 - name: "cicd"
-  component_path: "../../components/cicd"
+  component_path: "../../components/cicd/manual"
```

Comment:
I think the README.md should mention somewhere the split between manual/ and auto/, and how one is expected to use them (i.e. manually deploy stuff until manual, let CI/CD deploy stuff under auto/).

Reply:
Added a new README.md to describe the component. The README.md under manual/ is meant to show in the final cicd/ directory at root.